apps/test/unit/gamelab/GameLabSpriteTest.js

```diff
@@ -0,0 +1,63 @@
+var P5 = require("script!@cdo/apps/../lib/p5play/p5");
+var P5Play = require("script!@cdo/apps/../lib/p5play/p5.play");
```

Comment:
Since you don't ever call these directly in this scope, you can have the `require` statements without assigning their result:

```javascript
require("script!@cdo/apps/../lib/p5play/p5");
require("script!@cdo/apps/../lib/p5play/p5.play");
```